test/unit_test_impl.hpp

```diff
@@ -333,7 +333,14 @@ bool engine::run(bool colorize,
     // psv == pipe-separated-values
     strvec result;
     if (psv != ".*") {
-      split(result, psv, "|", token_compress_on);
+      // Quick & dirty fix
+      std::istringstream ss(psv);
+      std::string token;
+      while(std::getline(ss, token, ',')) {
```

Comment:
Style:
- space around keywords
- omit braces for single-line statements

Ideally we can use this clang-format style: https://github.com/actor-framework/actor-framework/blob/develop/.clang-format.